packages/core/src/portable.ts

```diff
@@ -13,9 +15,23 @@ export interface PortableResult {
 
 export interface PortableHint {
   requireFile: string;
+  packageVersion: string | undefined;
   useMethod?: string;
 }
 
+const { findUpPackagePath } = resolvePackagePath;
+
+export function maybeNodeModuleVersion(path: string) {
+  const packagePath = findUpPackagePath(path);
+
+  if (packagePath === null) {
+    // no package was found
+    return undefined; // should this bust the cache or ... ?
+  } else {
+    return readJSONSync(packagePath).version;
```

Comment:
Why not use `require(packagePath)` (vs `readJSONSync`)?

Reply:
linting yelled at me